### ISSUE TITLE
Please support the latest node 

### ISSUE THREAD

**Issue by dkwingsmt** [jeremyruppel/underscore.inflection#28]
Can you support the latest node version? It seems to work fine tho but the warning during ``npm i`` worries me a lot.

```
npm WARN engine underscore.inflection@1.3.0: wanted: {"node":">=0.10.0 <0.12"} (current: {"node":"5.2.0","npm":"3.3.12"})
```

**Comment by jeremyruppel** [comment]
@dkwingsmt I just released v1.3.1 which tests against the latest node versions and fixes that `npm install` warning. FYI the `engine` field is [only a warning]. Thanks!

[1]: https://docs.npmjs.com/files/package.json#engines

**Comment by dkwingsmt** [comment]
Thank you a lot! (and Merry Xmax
